### PR TITLE
Add code of conduct

### LINKED ISSUE #143: Code of conduct
As a community, it is great practice for us to have a code of conduct.

### PR DESCRIPTION
**Pull request by ewilderj**
Fixes #143 

I suggest we draw this to the attention of the list before finally merging. In particular, I want people to be OK with the conflict resolution body being the Google committee: this seems appropriate until such time as Kubeflow joins an external foundation. However, should the community want to form an alternate resolution committee, that would work as well.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Examples of unacceptable behavior by participants include:
```
Comment (jlewi) [inline CODE_OF_CONDUCT.md]: Markdown formatting error

"Examples" should not be an item in the current list and the next line "The use of sexualized" probably should be a list item.

Comment (ewilderj) [reply CODE_OF_CONDUCT.md]: Agreed. Looks like a copypasta error, thanks for catching!

**Comment by ewilderj** [comment]
@ScorpioCPH I think a community repo will make sense soon, when we have more to put in there. I suspect as soon as we start to form sub-groups, that will make sense. I see no difficulty with having the code reside in the main repo too.

**Comment by jlewi** [comment]
We'll discuss the creation of the community repository at this week's meeting. I think we need one for a lot of reasons.

**Comment by jlewi** [comment]
@ewilderj Do you want to close this PR and open one up against our community repo?

**Comment by ewilderj** [comment]
Closing in favor of https://github.com/kubeflow/community/pull/2 — if you were coming here to comment, please comment on that PR instead.

**Pull request closed without merging**